--- origsrc/openexr-1.6.1/IlmImf/Makefile.am	2007-03-30 12:14:05.000000000 -0500
+++ src/openexr-1.6.1/IlmImf/Makefile.am	2007-10-28 22:07:07.765625000 -0500
@@ -113,7 +113,7 @@
 b44ExpLogTable_SOURCES = b44ExpLogTable.cpp
 b44ExpLogTable_LDADD = @ILMBASE_LDFLAGS@ @ILMBASE_LIBS@
 
-b44ExpLogTable.h: b44ExpLogTable
+b44ExpLogTable.h: b44ExpLogTable$(EXEEXT)
 	./b44ExpLogTable > b44ExpLogTable.h
 
 BUILT_SOURCES = b44ExpLogTable.h
